modules/hcp-aws-deploy-hc.adoc

```diff
@@ -8,6 +8,18 @@
 
 You can deploy a hosted cluster on {aws-first} by using the `hcp` command-line interface (CLI).
 
+By default for {hcp} on {aws-first}, you use an AMD64 hosted cluster. However, you can enable {hcp} to run on an ARM64 hosted cluster. For more information, see "Running hosted clusters on an ARM64 architecture".
```

Comment:
We tend to avoid inline links because they add visual clutter and can unnecessarily pull the user away from the current task. I can add a link to that section of the docs in an "Additional resources" section at the end of this procedure.